Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -210,6 +215,8 @@ static DisplayType equivalentInlineDisplay(const RenderStyle& style)
     case DisplayType::TableCaption:
         return DisplayType::Inline;
 
+    case DisplayType::Math:
+        return display;
```

Comment:
This is correct, but probably should be grouped with the other "unmodified inline display" (`case DisplayType::RubyAnnotation`).